unix/vncpasswd/vncpasswd.cxx

```diff
@@ -165,24 +165,40 @@ int main(int argc, char** argv)
     FILE* fp = fopen(fname,"w");
     if (!fp) {
       fprintf(stderr,"Couldn't open %s for writing\n",fname);
+      delete [] fname;
+      delete obfuscated;
+      if (obfuscatedReadOnly)
```

Comment:
delete on nullptr does nothing (same as free) so this check should be unnecessary.